tests/unit/test_naming.py

```diff
@@ -31,10 +32,10 @@ def test_windows_reserved_names(self):
             'LPT5', 'LPT6', 'LPT7', 'LPT8', 'LPT9'
         }
 
-        for dirpath, dirnames, filenames in os.walk(self.esmvaltool_folder):
-            for dirname in dirnames:
-                if dirname in IGNORE:
-                    dirnames.remove(dirname)
+        for dirpath, dirnames, filenames in os.walk(self.esmvaltool_folder,
+                                                    topdown=True):
```

Comment:
`topdown=True` is the default option, I think you can remove that.

Reply:
well spotted! gonna remove :+1: